sinks/gcm/driver.go

```diff
@@ -191,6 +161,7 @@ type timeseriesDescriptor struct {
 type point struct {
 	Start       time.Time `json:"start,omitempty"`
 	End         time.Time `json:"end,omitempty"`
+	Int64Value  int64     `json:"int64Value"`
```

Comment:
Ok. Updated all metric to be doubles. The only downside is that now all the existing metrics are invalid. To solve this, I have added code to query existing metrics, check if they differ from requirements and delete if they do, and re-add the metric.